docs-source/content/samples/simple/elastic-stack/soa-domain/weblogic-logging-exporter-setup.md

```diff
@@ -0,0 +1,126 @@
+
+Copy the `weblogic-logging-exporter-0.1.1.jar` and `snakeyaml-1.23.jar` files to the domain home directory in the Administration Server pod.
+
+```
+$ kubectl cp <file-to-copy> <namespace>/<Administration-Server-pod>:<domainhome>
+
+```
+
+```
+$ kubectl cp snakeyaml-1.23.jar soans/soainfra-adminserver:/u01/oracle/user_projects/domains/soainfra/
+
+$ kubectl cp weblogic-logging-exporter-0.1.1.jar soans/soainfra-adminserver:/u01/oracle/user_projects/domains/soainfra/
+```
+
+#### Add a startup class to the domain configuration
+
+1. Using the WebLogic Server Administration Console, in the left-side navigation pane, expand **Environment**, and then select **Startup and Shutdown Classes**.
+
+1. Add a new startup class. You may choose any descriptive name, however, the class name must be `weblogic.logging.exporter.Startup`.   
+
+1. Target the startup class to each server from which you want to export logs.  
+
+You can verify this step by looking for the update in your `config.xml` file, which should look similar to the following example:
+```
+<startup-class>
+  <name>weblogic-logging-exporter</name>
+  <target>AdminServer,soa_cluster</target>
+  <class-name>weblogic.logging.exporter.Startup</class-name>
+</startup-class>
+```  
+
+#### Update the WebLogic Server `CLASSPATH`
+
+Copy the `setDomainEnv.sh` file from the pod to a local folder.  
+```
+$ kubectl cp soans/soainfra-adminserver:/u01/oracle/user_projects/domains/soainfra/bin/setDomainEnv.sh .
+```  
+
+Modify `setDomainEnv.sh` to update the server class path.  
+```
+CLASSPATH=/u01/oracle/user_projects/domains/soainfra/weblogic-logging-exporter-0.1.1.jar:/u01/oracle/user_projects/domains/soainfra/snakeyaml-1.23.jar:${CLASSPATH}
+export CLASSPATH
+```  
+
+Copy back the modified `setDomainEnv.sh` file to the pod.  
+```
+$ kubectl cp setDomainEnv.sh soans/soainfra-adminserver:/u01/oracle/user_projects/domains/soainfra/bin/setDomainEnv.sh
+```
+
+#### Create a configuration file for the WebLogic Logging Exporter  
+
+Create a file named `WebLogicLoggingExporter.yaml`. Specify the Elasticseach server host and port number.
+```
+weblogicLoggingIndexName: wls
+publishHost: elasticsearch.default.svc.cluster.local
+publishPort: 9200
+domainUID: soainfra
+weblogicLoggingExporterEnabled: true
+weblogicLoggingExporterSeverity: TRACE
+weblogicLoggingExporterBulkSize: 1
+```  
+
+Copy the `WebLogicLoggingExporter.yaml` file to the domain home directory in the WebLogic Administration Server pod.  
+```
+$ kubectl cp WebLogicLoggingExporter.yaml soans/soainfra-adminserver:/u01/oracle/user_projects/domains/soainfra/config/
+```  
+
+#### Restart all the servers in the domain
+
+To restart the servers, stop and then restart them using the following commands.
+
+To stop the servers:
+```
+$ kubectl patch domain soainfra -n soans --type='json' -p='[{"op": "replace", "path": "/spec/serverStartPolicy", "value": "NEVER" }]'
+
+```
+
+To restart the servers:
```

Comment:
In WebLogic, restart is a single operation (which stops and then starts the server). Below command only starts the servers (if they are already stopped), however it will not restart (stop+start) the servers. Please see if it make sense to specify 
"To start the servers:"  
Similarly at line 105 also, "stop and then start".

Reply:
Yes, got it. I changed it in both places.